mythril/mythril.py

```diff
@@ -36,8 +36,11 @@
 from mythril.support.truffle import analyze_truffle_project
 from mythril.ethereum.interface.leveldb.client import EthLevelDB
 from mythril.laser.smt import SolverStatistics
+from mythril.start_time import StartTime
+
 
 log = logging.getLogger(__name__)
+start_time = StartTime()
```

Comment:
I'm thinking that we should maybe re-initialize the start time for each contract that is being analyzed, wdyt?

Reply:
what was I thinking when I put it there :man_facepalming: , will move it there :+1: